app/routes/boxes.py

```diff
@@ -53,10 +53,19 @@ def start_box() -> Tuple[Response, int]:
 
         config_id = dig(request.json, "data/relationships/config/data/id")
         ssh_key = dig(request.json, "data/attributes/sshKey")
+        image = dig(request.json, "data/attributes/image", "ubuntu")
+
+        #right now there is only one valid choice
+        #in the future there will be multiple choices and customer snapshots
+        #we will want to throw an error in the future for an invalid choice
+        if image == "ubuntu":
```

Comment:
This is the sort of thing that would go in the config, actually - since it might be different per launch.

Reply:
I was thinking this.  Right now, they just need to pass in one thing, what they want to run (distro or app) and that will directly related to one of our images.  This is allowed to be unique per launch, because it is within the json sent to the api call.  In the future if they provide a config name, we will look up the details of that custom config instead.  Make sense?